### ISSUE TITLE
1.0.1: broken sdist tar ball.

### ISSUE THREAD

**Issue by kloczek** [asadmoosvi/pypi-search#2]
Looks like some files are missing in sdist pypi tar ball.
```console
+ /usr/bin/python3 setup.py build '--executable=/usr/bin/python3 -s'
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    with open('requirements.txt') as req_f:
FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```

**Comment by asadmoosvi** [comment]
Fixed in d77a58e.

**Comment by kloczek** [comment]
Thank you. It works now.
